README.md

```diff
@@ -126,14 +126,17 @@ Tarantool connectors for various programming languages.
 - [PHP PECL connector](https://github.com/tarantool/tarantool-php).
 - [PHP pure connector](https://github.com/tarantool-php/client).
 - [Python connector](https://github.com/tarantool/tarantool-python).
-- [Java connector](https://github.com/tarantool/tarantool-java).
+- [Java connector (old)](https://github.com/tarantool/tarantool-java) -- deprecated, but offers JDBC interface for a single Tarantool node and supports ancient Tarantool versions (1.7+).
+- [Java driver](https://github.com/tarantool/cartridge-java) -- asynchronous driver based on Netty framework, supports both single Tarantool nodes and clusters (1.10+), as well as applications using Cartridge framework and its modules.
```

Comment:
Both connectors are asynchronous. Let's say it for both or neither, otherwise it would look confusing.

Reply:
Why not just keep the most advanced one (espessially if the other one is "old")? All in all, this is an "awesome" list, not an index of all available connectors/modules/tools.

Reply:
That's correct point. The similar idea is stated in the [CONTRIBUTING.md](https://github.com/tarantool/awesome-tarantool/blob/8e4ab0b1d4596e94bca6f83105374460e4dce794/CONTRIBUTING.md) document:

> What this list doesn't need or have much room for:
> <...>
> * Packages which have a strictly better alternative available, or which don't add to the picture of what's out there.

To be honest, my dream is to have solid ecosystem around tarantool. Now it is in the weird state: several connectors for the same language, what a hell? And, to be honest and precise, I don't like how cartridge-java appears: instead of evolving tarantool-java, refactor it, deprecate bad APIs (in sync with major release version bumps) it becomes the competitor. It was okay for a prototype, then we should going with some 'evolution' plan, not 'revolution' one.

Anyway, now we're in the world, where both exist (the similar stories occur with Python and Go connectors too).

What stop me from say 'tarantool-java is finally gone' is JDBC support (and really lot of work we pay there)? The Idea plugin, which recently was included into the list, uses this old tarantool-java connector just because of JDBC.

I'll cite the CONTRIBUTING.md again:

> Packages that do a specific, niche sort of thing that few people will use.

I guess single node JDBC fits there. So it should be okay to remove it from the list.

BTW, we plan to perform deep comparison of state of popular connectors, especially, where several alternatives exist, and pay significant time toward the solid ecosystem (deprecations or resurrections — will heavily depend on particular situation).

Reply:
> Let's say it for both or neither, otherwise it would look confusing.

Removed "asynchronous".

> Originally, I wrote about "no active maintenance" for tarantool-java

Fixed.

Reply:
> So, I vote for both connectors to be listed here with correct stating the differences between them.

If the "old" connector still does its job and has features that the other one doesn't, then maybe it shouldn't be called "old"?  What confuses me the most is not that there will be 2 connectors, but that one of them is "old" and "deprecated". From my experience with other "awesome" lists, deprecated projects are usually removed from the list, not added :)

Reply:
@rybakit By naming it "old" I meant "the elder":) It is hard to distinguish them by names otherwise. Probably we need to name one of them a "JDBC single-node driver" and the another a "cluster driver", let's discuss it.

Reply:
Renamed `tarantool-java` into "JDBC driver".

Reply:
Now the descriptions are okay for me. I'll resolve the discussion.